projects/aca-content/src/lib/dialogs/node-template/create-from-template.dialog.scss

```diff
@@ -23,6 +23,7 @@
 
   .mat-form-field {
     margin-bottom: 20px;
+    width: 100%;
```

Comment:
should not we avoid referring to external library classes?

Reply:
it's the problem we have with the Angular upgrade, should not be touching internal classes of material

Reply:
thanks, done